src/effects/VST/VSTEffect.cpp

```diff
@@ -804,10 +804,15 @@ VSTEffect::VSTEffect(const PluginPath & path, VSTEffect *master)
    mTimeInfo.timeSigDenominator = 4;
    mTimeInfo.flags = kVstTempoValid | kVstNanosValid;
 
-   // If we're a slave then go ahead a load immediately
+   // Get the constant data that are shared by all instances
+   // (e.g. entry point in the dll, vst version, no. of channels etc.)
+   LoadCommon();
```

Comment:
When `LoadCommon` is lifted out of `Load`, then you might `move` its return value into a new argument of `Load` to transfer ownership of the handle.